src/main/java/it/gov/pagopa/pu/workflow/wf/ingestionflow/paymentsreporting/activity/NotifyPaymentsReportingToIufClassificationActivity.java

```diff
@@ -1,14 +1,13 @@
 package it.gov.pagopa.pu.workflow.wf.ingestionflow.paymentsreporting.activity;
 
 import io.temporal.activity.ActivityInterface;
-import it.gov.pagopa.payhub.activities.dto.classifications.Transfer2ClassifyDTO;
+import it.gov.pagopa.payhub.activities.dto.classifications.PaymentsReportingTransferDTO;
 
 import java.util.List;
 
 @ActivityInterface
 public interface NotifyPaymentsReportingToIufClassificationActivity {
 
-  void signalIufClassificationWithStart(Long organizationId, String iuf,
-                                        String outComeCode, List<Transfer2ClassifyDTO> transfers2classify);
+  void signalIufClassificationWithStart(String iuf, List<PaymentsReportingTransferDTO> transfers);
```

Comment:
let's keep the organizationId between the parameters, it will be the same for all the transfers